README.txt

```diff
@@ -0,0 +1,101 @@
+### FOLDER STRUCTURE ###
+
+In order to run the following script it is important to ensure, that the following folder structure is maintained.
+The data directories train and test can be omitted, when the model is only used for inference.
+
+|-- code
+    |-- data
+        |-- train
+            <1st focal plane image>.png
+            ...
+            <nth focal plane image>.png
+            <ground truth image>.png
+            ...
+
+        |-- test
+            <1st focal plane image>.png
+            ...
+            <nth focal plane image>.png
+            <ground truth image>.png
+            ...
+
+        |-- inference
+            <1st focal plane image>.png
+            ...
+            <nth focal plane image>.png
+            <ground truth image>.png
+            ...
+
+    |-- model
+        model.pt
```

Comment:
i think this is the wrong place. check when the upload instructions are visible on moodle again.